english/glowing-pants/dance-dance/dance-dance.md

```diff
@@ -0,0 +1,5 @@
+You dance, and the friction of your inspired moves begins to create heat.
+
+Heat, of course, melts mashmallows, which poses a considerable challenge.  
+
+[Stop dancing before you melt the place and recommence contemplating your predicament](/../..)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.